document_creation.py

```diff
@@ -0,0 +1,21 @@
+import os
+import csv
+
+
+class file_create:
```

Comment:
In this case, file_create doesn't have state(no fields, not data) so we don't need to have it as a class. Let's rather declare pure functions and import them in place.

Reply:
Let's name it create_directory? Usually function names are verbs.

Reply:
Will do!